src/_data/config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "maxPostsInFeed": 10,
+  "languages": [
+    {
+      "code": "en-CA",
+      "dir": "ltr",
+      "name": "English"
+    },
+    {
+      "code": "fr-CA",
+      "dir": "ltr",
+      "name": "Français"
+    }
+  ],
```

Comment:
Is there a specific reason to define this as an array instead of an object? If it was an object you could key off of the language code, as you do in other places like site.json and translations.json. Also you probably wouldn't need `getSiblingValue` anymore.

Reply:
Yes— this makes it editable in the CMS.

Reply:
Can you expand on this a bit more. What's the restriction? Does the CMS consume the config.json file directly and requires an array?